packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -52,18 +52,46 @@ import { TimesIcon, PlusCircleIcon } from '@patternfly/react-icons';
 
 LinkButton = () => (
   <React.Fragment>
-    <Button component="a" href="https://pf-next.com/" target="_blank" variant="primary">
-      Link to Core Docs
+    <Button component="a" href="https://pf4.patternfly.org/" target="_blank" variant="primary">
+      Link to core docs
     </Button>{' '}
-    <Button component="a" href="https://pf-next.com/" target="_blank" variant="secondary">
-      Secondary Link to Core Docs
+    <Button component="a" href="https://pf4.patternfly.org/" target="_blank" variant="secondary">
+      Secondary link to core docs
     </Button>{' '}
-    <Button component="a" isDisabled href="https://pf-next.com/" target="_blank" variant="tertiary">
-      Tertiary Link to Core Docs
+    <Button component="a" isDisabled href="https://pf4.patternfly.org/" target="_blank" variant="tertiary">
+      Tertiary link to core docs
     </Button>
     <Button component="a" href="https://pf4.patternfly.org/contribution/#modifiers" variant="link">
       Jump to modifiers in contribution guidelines
     </Button>
   </React.Fragment>
 );
 ```
+
```

Comment:
It might we worth adding a note in the docs here about when a consumer would want/need to use the inoperable state over the disabled state.

Reply:
I guess the example having a tooltip demonstrates that, but looking at that in the docs doesn't immediately make it clear that a tooltip won't work on a disabled button.

Reply:
Good point - I made this more clear using the [tooltip text](https://github.com/patternfly/patternfly-react/pull/3801/files#diff-5ef4ad66c44ff42efa5f8359d5ccce44R77)